www/assets/css/course-collection.scss

```diff
@@ -25,18 +25,70 @@
   }
 }
 
+h1.course-list-title {
+  color: $highlight-red;
+  text-transform: none;
+  font-size: $heading-two-font-lg;
```

Comment:
I think we can remove this font-size as `$font-xl` is already being applied below or vice versa.

Reply:
So $heading-two-font-lg is 28 px
But $font-xl is 24px
I needed the top title to be bigger than the list titles

Reply:
Oh wait there's 2 of the same font size in the same class
Fixing!

Reply:
Other than this, I think we are good!
If @Ferdi is okay with the design

Reply:
Let's merge it . thank you both !